eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AfkService.java

```diff
@@ -41,28 +52,57 @@ public boolean isAfk(UUID player) {
     }
 
     public void markInteraction(UUID player) {
-        int interactions = this.interactions.getOrDefault(player, 0);
-
-        interactions++;
+        int count = this.interactionsCount.getOrDefault(player, 0);
+        count++;
 
-        if (interactions >= this.afkSettings.interactionsCountDisableAfk()) {
+        if (count >= this.afkSettings.interactionsCountDisableAfk()) {
             this.clearAfk(player);
             return;
         }
 
-        this.interactions.put(player, interactions);
+        this.interactionsCount.put(player, count);
     }
 
-    public boolean clearAfk(UUID player) {
+    public void clearAfk(UUID player) {
```

Comment:
dodać czyszczenie mapki z Instant, on quit będzie to ładnie czyściło dane gracza.